addons/mail/models/mail_rtc_session.py

```diff
@@ -0,0 +1,141 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from dateutil.relativedelta import relativedelta
+
+from odoo import api, fields, models
+
+
+class MailRtcSession(models.Model):
+    _name = 'mail.rtc.session'
```

Comment:
Namespacing is weird. Nothing to do with standard mail, especially with split in mind. Would go for ``mail.channel.something`` as this is completely linked to new channels management.